cilium-cli/utils/features/features.go

```diff
@@ -184,8 +187,9 @@ type Requirement struct {
 	requiresEnabled bool
 	enabled         bool
 
-	requiresMode bool
-	mode         string
+	requiresMode     bool
+	requireModeIsNot bool
```

Comment:
nit (non-blocking, feel free to ignore): I wonder if we should introduce an `enum` here, rather than having three enums (`requiresEnabled`, `requiresMode`, `requiresModeIsNot`)

```go
type requirementMatchType = string

const (
  requiresEnabled requirementMatchType = "enabled"
  requiresDisabled = "disabled"
  requiresModeEqual = "equal"
  requiresModeNotEqual = "notEqual"
)
```

And then replace this here with
```
match requirementMatchType
```

Reply:
Does sound like a good idea. We can do this in a follow up PR.